aikau/src/main/resources/alfresco/services/LightboxService.js

```diff
@@ -19,16 +19,16 @@
 
 /**
  * @module alfresco/services/LightboxService
- * @extends module:alfresco/core/Core
+ * @extends module:alfresco/services/BaseService
  * @author Kevin Roast
  */
 define(["dojo/_base/declare",
-        "alfresco/core/Core",
+        "alfresco/services/BaseService",
         "dojo/_base/lang",
         ""],
```

Comment:
Bugger, I removed it and then discarded changes and forgot to remove it again... this is why we have these nice code reviews :)